The format for the `date` command has changed and this test wasn't updated with it.

--- a/testsuite/date/date-works
+++ b/testsuite/date/date-works
@@ -1,6 +1,6 @@
 dt=`busybox date`
 # Expected format: Fri Apr 25 03:47:55 CEST 2008
-dt=`echo "$dt" | sed 's/^[^ ][^ ][^ ] [^ ][^ ][^ ] [ 0123][0-9] [012][0-9]:[0-5][0-9]:[0-6][0-9] [A-Z][A-Z]* [012][0-9][0-9][0-9]$/OK/'`
+dt=`echo "$dt" | sed 's/^[^ ][^ ][^ ] [^ ][^ ][^ ] [ 0123][0-9] [012][0-9]:[0-5][0-9]:[0-6][0-9] -[0-9][0-9]* [012][0-9][0-9][0-9]$/OK/'`
 test x"$dt" = x"OK"
 
 dt=`busybox date -d 1:2`
